src/xlsx.js

```diff
@@ -0,0 +1,130 @@
+export class Workbook {
+  constructor(workbook) {
+    Object.defineProperties(this, {
+      _: {value: workbook},
+      sheetNames: {
+        value: workbook.worksheets.map((sheet) => sheet.name),
+        writable: false,
+        enumerable: true,
+      },
+    });
+  }
+  sheet(name, {range, headers = false} = {}) {
+    const sname =
+      typeof name === "number"
+        ? this.sheetNames[name]
+        : this.sheetNames.includes(name + "")
+        ? name + ""
```

Comment:
Nit: Coerce once to a string once to avoid potentially nondeterministic toString?

```suggestion
        : this.sheetNames.includes(name += "")
        ? name
```